src/components/Tooltip/Tooltip.jsx

```diff
@@ -77,7 +77,7 @@ class Tooltip extends React.Component {
   /**
    * Rendering label with hotkey option if available
    */
-  renderLabel = (label, hotkey) => (
+  renderLabel = (label, hotkey, customHTML) => (
```

Comment:
It might be good to set `customHTML = null` as a default value, then null-ish values will be more likely